### PR TITLE
Error rate calculation for report

### PR DESCRIPTION
**Pull request by doorooful**
- Measure original file size
- Get size of error data
- Calculate error rate and render on report

### CODE REVIEW

**Comment by doorooful** [comment]
- Result: showing calculated error rate

Screenshot 2023-09-24 at 5 08 38 PM

File: dt-demo-be/app.py [hunk 1]
```diff
@@ -111,6 +111,26 @@ def get_overview_numbers():
...
+def get_overview_rate():
+    db_path = os.path.join("..", "database")
+
+    # Calculate overall
+    original_file_path = os.path.join(db_path, "demo.txt")
+    with open(original_file_path, 'rb') as file:
+        file_contents = file.read()
+    original_file_size = len(file_contents)
+    file.close()
+
```
Comment (leeinsunny) [inline dt-demo-be/app.py:123]: Can you add comment about calculation?

**Review by leeinsunny — CHANGES_REQUESTED** [review]
Please refer to my comment:)

**Review by leeinsunny — APPROVED** [review]
Thank you for your effort!

**Pull request merged**